### DISCUSSION TITLE
how can i run whisper on CPU faster, It's taking too much time

### DISCUSSION THREAD

**Discussion by tirth26patel** [openai/whisper#2278] [General]
for example, by using whisper on CPU it's taking too much time for transcribing a 30 second audio and i have also tried this using python the results are same

**Comment by glangford** [comment]
For just 30s of audio much of the time can be spent just loading the model. Ideally you want to load the model once, then reuse it across many files. 

Other suggestions:
* use a smaller model (for example medium is about 2x faster than large, see https://github.com/openai/whisper#available-models-and-languages
* quantize the model as suggested here https://github.com/openai/whisper/discussions/1990
